schema/taskfile-schema.json

```diff
@@ -107,7 +107,7 @@
           "method": {
             "description": "Defines which method is used to check the task is up-to-date. timestamp will compare the timestamp of the sources and generates files. checksum will check the checksum (You will probably want to ignore the .task folder in your .gitignore file). none skips any validation and always run the task",
             "type": "string",
-            "enum": ["none", "checksum", "timestamp"],
+            "enum": [ "none", "checksum", "timestamp" ],
```

Comment:
I think these additional spaces are uncommon as a convention in the JSON/JavaScript world. No strong opinion, but I think I'd keep as before.

Reply:
Yeah, that's fine. These were added by whatever formatter I have set up for JSON files. I'll disable this for the Task repo in the future.